docs/quickstart.md

```diff
@@ -5,54 +5,11 @@ The purpose of this guide is to install a minimal local Armada deployment for te
 ## Pre-requisites
 
 - Git
-- Docker
+- Docker (Docker Desktop recommended for local development)
```

Comment:
I don't use docker desktop on linux, and wouldn't, because it adds a VM layer that hurts performance and needs separate management (e.g. allocating it more ram/cpus as needed) -- in fact the install guide dictates you configure KVM before doing this, which makes it a nonstarter on systems where folks are running another hypervisor software (like vmware/virtualbox).

I do agree that Docker Desktop makes the most sense on Mac and Windows, though.

Reply:
Also note that Docker Desktop has built-in K8S support (at least, on macOS) - turn it on via a checkbox in the config panel, then restart Docker Desktop - this may run more reliably than Kind-managed clusters over Docker.